data/eos-shell-extension-prefs.desktop.in.in

```diff
@@ -1,6 +1,6 @@
 [Desktop Entry]
 Type=Application
-_Name=GNOME Shell Extension Preferences
+_Name=Extensions
```

Comment:
Seeing an icon that says just “Extensions”, they will wonder what the extensions are for. Maybe we should consider using ”Desktop extensions” instead, dunno.